otx/cli/builder/builder.py

```diff
@@ -174,15 +180,28 @@ def build_task_config(self, task_type, model_type=None, workspace_path=None, otx
         template_config = MPAConfig.fromfile(template.model_template_path)
         template_config.hyper_parameters.base_path = "./configuration.yaml"
 
+        # Configuration of Train Type value
+        sub_rel_path = "."
+        if train_type.lower() != "incremental":
+            sub_rel_path = train_type.lower()
```

Comment:
We already get `train_type` as lower case at [link](https://github.com/openvinotoolkit/training_extensions/pull/1485/files#diff-bf8911e3241497afed1050a4d4a1995b8dc64417cc0edfaca70668d32dc72421R35).
I think `.lower()` is a duplicated operation.

Reply:
yes i will fix it! thanks

Reply:
Fixed! https://github.com/openvinotoolkit/training_extensions/pull/1485/commits/598606871255bd2ceeddcaecec3ed016f64b198a